### DISCUSSION TITLE
New Source: Selligent

### DISCUSSION THREAD

**Discussion by awoehrl** [airbytehq/airbyte#33177] [New Connector Request]
## Tell us about the new integration you’d like to have
Which source and which destination? Which frequency?
Selligent could be a source and destination.

## Describe the context around this new integration
Which team in your company wants this integration, what for? This helps us understand the use case.
Extracting marketing data to our data warehouse

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
FTP exports probably

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368201171133) by [Unito](https://www.unito.io)

**Comment by cgardens** [comment]
thanks @awoehrl ! we will add it to our list of upcoming integrations.

**Comment by cgardens** [comment]
Would you be willing to tell us a bit more about your use case? I am not terribly familiar with Selligent, so if you have any more specific guidance on what data you'd be most interested in extracting from Selligent that would be great. Would Selligent as a source be enough to get you going or do you need it as both a source and a destination?

**Comment by sherifnada** [comment]
@awoehrl we're going to apply for a test account from Selligent. Could we mention you or your company as a "mutual customer" of Airbyte & Selligent? It helps with the account creation process.

**Comment by johnlafleur** [comment]
Just did a call with them to get the sandbox account. We should have some news beginning of January 2021

**Comment by karinakuz** [comment]
There is no open information for Vetting about API and integration. Account is not free and available only for the Partner. Sent request on the site describing our goals and asking for a sandbox account https://www.selligent.com/register-to-partner?ID=Pnkp%2Bfo9ZoEWQwpOSTUgGkUMPB8JaJk3my6NOv1k_icQQEXF7XTVpuH18kZnK8HXy7uVuKptr2wG10RPPG

**Comment by karinakuz** [comment]
Sent several letters in October (contact@ner.selligent.com) but there is no answer from Selligent

[a comment by an automated account is not shown]
